Content.Server/Atmos/EntitySystems/FlammableSystem.cs

```diff
@@ -494,21 +495,35 @@ public override void Update(float frameTime)
 
                     var ev = new GetFireProtectionEvent();
                     // let the thing on fire handle it
-                    RaiseLocalEvent(uid, ref ev);
-                    // and whatever it's wearing
+                    RaiseLocalEvent(uid, ref ev);                    // and whatever it's wearing
```

Comment:
Don't need to change the comment position
```suggestion
                    RaiseLocalEvent(uid, ref ev);
                    // and whatever it's wearing
```